source/node-reference/node-reference.rst

```diff
@@ -22,9 +27,12 @@ List with all the nodes
    nodes/DepthMapFilter
    nodes/ExportAnimatedCamera
    nodes/ExportColoredPointCloud
+   nodes/ExportMatches
    nodes/ExportMaya
    nodes/FeatureExtraction
    nodes/FeatureMatching
+   nodes/GlobalSfM
+   nodes/HDRIstitching
```

Comment:
This has been removed. We now use "Panorama*" nodes.

Reply:
So panoramaInit, panoramaEstimation, panoramaWarping, panoramaCompositing ...